LastInfoSec/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7.4-alpine3.10
+
+# Copy the connector
+COPY src /opt/opencti-connector-lastinfosec-cti
```

Comment:
If there will be no further connector maybe just stick with `opencti-connector-lastinfosec` ?
If okay: replace in all files :)

Reply:
Agree.

Reply:
Fixed :)